org.eclipse.scanning.api/src/org/eclipse/scanning/api/device/IRunnableDeviceService.java

```diff
@@ -184,20 +184,20 @@
      * @return
      */
 	Collection<DeviceInformation<?>> getDeviceInformation() throws ScanningException;
-	
+
     /**
      * Get the information for all the runnable devices currently created with a specific role.
      * Will not get device information that is potentially held on the device if the device is not alive.
      * @return
      */
 	Collection<DeviceInformation<?>> getDeviceInformation(DeviceRole role) throws ScanningException;
-	
+
     /**
      * Get the information for the named runnable device.
      * @return
      */
 	DeviceInformation<?> getDeviceInformation(String name) throws ScanningException;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1452)